packages/discovery/README.md

```diff
@@ -0,0 +1 @@
+# @substrate/discovery
```

Comment:
Maybe populate this with something small to begin with (we can improve later) like:

This package implements the discovery protocol that browsers use to find compliant browser extensions. 

The main export is a function called `getProviders`. This function dispatches an event on the `window` object that compliant browser extensions (or similar) may respond to by providing back an interface of the correct shape. An array of all such interfaces that we get back will be given back to the caller of `getProviders`.